jcl/src/java.base/share/classes/java/lang/invoke/ArrayVarHandle.java

```diff
@@ -119,10 +119,6 @@
 	}
 /*[ENDIF] Java12 */
 
-	public MethodType accessModeTypeUncached(AccessMode accessMode) {
-		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
-	}
```

Comment:
This is changed from `public` to package access which might be problematic.

Reply:
https://download.java.net/java/early_access/jdk16/docs/api/java.base/java/lang/invoke/VarHandle.html

According to the above Java doc, `VarHandle.accessModeTypeUncached` is not public. OpenJ9 classes, which extend the abstract `Varhandle` class, made it public but those OpenJ9 classes are not subject to the JCK `api.signaturetest`. Making `accessModeTypeUncached` public in the abstract `VarHandle` class will break the JCK `api.signaturetest`. So, we should keep `VarHandle.accessModeTypeUncached` package-protected as per the current changes.

Reply:
https://download.java.net/java/early_access/jdk16/docs/api/java.base/java/lang/invoke/VarHandle.html

According to the above Java doc, `VarHandle.accessModeTypeUncached` is not public. OpenJ9 classes, which extend the abstract `Varhandle` class, made it public but those OpenJ9 classes are not subject to the compliance (signature) test. Making `accessModeTypeUncached` public in the abstract `VarHandle` class will break the compliance (signature) test. So, we should keep `VarHandle.accessModeTypeUncached` package-protected as per the current changes.